packages/core/src/events/DefaultEventHandlers.ts

```diff
@@ -245,7 +245,7 @@ export class DefaultEventHandlers<O = {}> extends CoreEventHandlers<
       },
       create: (
         el: HTMLElement,
-        userElement: React.ReactElement,
+        userElement: React.ReactElement | (() => NodeTree),
```

Comment:
Could we have `userElement: React.ReactElement | NodeTree` instead?

In that case, we will probably need a helper function (ie: `isNodeTree`) to check if the input is a valid `NodeTree` as well.

Reply:
You cannot replace it with NodeTree.Because this function needs to be executed each time to generate a new ID for each node.If it is NodeTree, the id of each drag will be the same